index.bs

```diff
@@ -11117,48 +11118,82 @@ is the concatenation of the [=interface=]’s
+            \[[Value]]: |value|}.
+        1.  Let |id| be |const|'s [=identifier=].
+        1.  Perform [=!=] <a abstract-op>DefinePropertyOrThrow</a>(|target|, |id|, |desc|).
+</div>
 
 
 <h4 id="es-attributes">Attributes</h4>
 
-For each [=exposed=] [=attribute=] of the [=interface=]
-there must exist a corresponding property.
-The characteristics of this property are as follows:
+[=Static attributes=] are exposed of the [=interface object=].
+[=Regular attributes=] are exposed on the [=interface prototype object=],
+unless the attribute is [=unforgeable=] or if it was declared with the [{{Global}}] [=extended attribute=],
+in which case they are exposed on every object that implements the interface.
 
-*   The name of the property is the [=identifier=] of the [=attribute=].
-*   The location of the property is determined as follows:
-    *   If the attribute is a [=static attribute=],
-        then there is a single corresponding property and it exists on the interface’s [=interface object=].
-    *   Otherwise, if the attribute is [=unforgeable=] on
-        the interface or if the interface was declared with the [{{Global}}] [=extended attribute=],
-        then the property exists on every object that implements the interface.
-    *   Otherwise, the property exists solely on the interface’s [=interface prototype object=].
-*   The property has attributes { \[[Get]]: |G|, \[[Set]]: |S|, \[[Enumerable]]: <emu-val>true</emu-val>, \[[Configurable]]: |configurable| },
-    where:
-    *   |configurable| is <emu-val>false</emu-val> if the attribute was
-        declared with the [{{Unforgeable}}] extended attribute and <emu-val>true</emu-val> otherwise;
-    *   |G| is the [=attribute getter=] created given the attribute, the interface,
-        and the [=relevant Realm=] of the object that is the location of the property; and
-    *   |S| is the [=attribute setter=] created given the attribute, the interface,
-        and the [=relevant Realm=] of the object that is the location of the property.
+<div algorithm>
+    To <dfn>expose the regular attributes</dfn> of [=interface=] |interface| on |target|, given [=Realm=] |realm|, 
+    run the following steps:
+
+    1.  Let |attribues| be the [=list=] of [=exposed=] [=regular attributes=] that are [=members=] of |interface|.
+    1.  [=list/Remove=] from |attribues| all the [=attribues=] that are [=unforgeable=].
+    1.  [=Expose the attributes=] |attributes| of |interface| on |target| given |realm|.
+</div>
+
+<div algorithm>
+    To <dfn>expose the static attributes</dfn> of [=interface=] |interface| on |target|, given [=Realm=] |realm|, 
+    run the following steps:
+
+    1.  Let |attribues| be the [=list=] of [=exposed=] [=static attributes=] that are [=members=] of |interface|.
+    1.  [=Expose the attributes=] |attributes| of |interface| on |target| given |realm|.
+</div>
+
+<div algorithm>
```

Comment:
This seems to be unused. I guess it could be done in the constructor ("interface object").